src/bundle/Command/DumpSqlCommand.php

```diff
@@ -27,6 +25,9 @@
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Style\SymfonyStyle;
 
+/**
+ * @template-covariant T of \Doctrine\DBAL\Platforms\AbstractPlatform
```

Comment:
Where is this type used?

Reply:
> Where is this type used?

Was used at some point, but after further refactoring it's no longer needed. Good catch, thanks.